boot/bootutil/src/loader.c

```diff
@@ -2270,11 +2270,11 @@ context_boot_go(struct boot_loader_state *state, struct boot_rsp *rsp)
 #ifdef MCUBOOT_VALIDATE_PRIMARY_SLOT
 #ifdef PM_S1_ADDRESS
 	/* Patch needed for NCS. Image 1 primary is the currently
-	 * executing MCUBoot image, and is therefore already trusted and
-	 * does not need validation.
+	 * executing MCUBoot image, and is therefore already validated by NSIB and
+	 * does not need to also be validated by MCUBoot.
 	 */
-    bool image_needs_validation = BOOT_CURR_IMG(state) != 1;
-	if (image_needs_validation)
+    bool image_validated_by_nsib = BOOT_CURR_IMG(state) == 1;
```

Comment:
Weird indentation?

Reply:
There is spaces here. Change to Tab

Reply:
I changed to spaces as that was what was used by surrounding code.

![image](https://user-images.githubusercontent.com/609230/207301478-51736e9e-0ba1-4674-9053-d2e741c56c6a.png)

Reply:
Fair enough.